tests/components/yamaha_musiccast/conftest.py

```diff
@@ -0,0 +1,98 @@
+"""Configuration for yamaha_musiccast tests."""
+from unittest.mock import patch
+
+from aiomusiccast import MusicCastConnectionException
+import pytest
+
+
+@pytest.fixture(autouse=True)
+async def autouse_mock_ssdp(mock_ssdp):
+    """Auto use mock_ssdp."""
+    yield
+
+
+@pytest.fixture(autouse=True)
+def mock_setup_entry():
```

Comment:
This is changing the scope of this fixture. Please keep it in the config flow module. Please keep all the existing fixtures in their place unless there's a bug with that.

Reply:
Changed.

Though, shouldn't the silent ssdp-fixtures be globally applied? The test environment can be different (local, in container, during CI, ...) potentially resulting in different outcomes. I.e., when working on the `sonos` component, one usually has a sonos-device, which will respond to search requests when sent, most likely interfering with the test itself.

Or, which was applied recently, the fixture `mock_get_source_ip`. Multi-homed environments resulted in failing tests. Applying this globally results in a predictable environment, saving rework later on. It's the implicit vs explicit debate of course.

Reply:
I think each integration needs to decide when to apply test patches.